llvm/test/MachineVerifier/test_g_vscale.mir

```diff
@@ -1,4 +1,5 @@
-# RUN: not --crash llc -verify-machineinstrs -run-pass none -o /dev/null %s 2>&1 | FileCheck %s
+# RUN: not --crash llc -verify-machineinstrs -mtriple=arm64 -run-pass none -o /dev/null %s 2>&1 | FileCheck %s
+# REQUIRES: aarch64-registered-target
```

Comment:
Why do we need to add target and requires? 

https://github.com/llvm/llvm-project/pull/84542#discussion_r1518471217

Reply:
I knew that this was coming. `ninja check-llvm-machineverifier` was complaining that the `llc` s had no target.

Reply:
Was this occurring prior to this patch?

Reply:
IDK. The first time, I called `ninja check-llvm-machineverifier`. This is my first instruction.

Reply:
Note that, I register exactly AArch64 and AMDGPU.

Reply:
I got:
```
llvm-project2/bin/llc: error: unable to get target for '', see --version and --triple
```

Reply:
```
********************
********************
Failed Tests (3):
  LLVM :: MachineVerifier/test_g_extract_subvector.mir
  LLVM :: MachineVerifier/test_g_insert_subvector.mir
  LLVM :: MachineVerifier/test_g_vscale.mir


Testing Time: 1.52s

Total Discovered Tests: 98
  Unsupported      : 15 (15.31%)
  Passed           : 77 (78.57%)
  Expectedly Failed:  3 (3.06%)
  Failed           :  3 (3.06%)
FAILED: test/CMakeFiles/check-llvm-machineverifier
```

Reply:
Every MachineVerifier test needs a REQUIRES, simply because we haven't bothered to add target specific subdirectories with the appropriate lit.local.cfg in each

Reply:
I believe the default target is still the host target, which I'm assuming is x86. You would have to manually override to set one. The real condition here is probably REQUIRES: default_triple